features/step_definitions/logging.rb

```diff
@@ -705,3 +705,135 @@
     end
   end
 end
+
+# This step will deploy one kafka cluster and create 4 topics(topic-logging-all,topic-logging-infra,topic-logging-app,topic-logging-audit) in this kafka
+Given /^I deployed kafka in the #{QUOTED} project via amqstream operator$/ do | project_name|
+  ensure_admin_tagged
+  step %Q/I switch to cluster admin pseudo user/
+  step %Q/I use the "#{project_name}" project/
+  step %Q/I process and create:/, table(%{
+    | f | #{BushSlicer::HOME}/testdata/logging/clusterlogforwarder/kafka/amq/02_og_amqstreams_template.yaml |
+    | p | AMQ_NAMESPACE=#{project_name} |
+  })
+  raise "Error create operatorgroup" unless @result[:success]
```

Comment:
I don't know if the step will work or not as I don't find where the `@result` come from. I suggest to use `step %Q/the step should succeed/`